interface/forms/sdoh/new.php

```diff
@@ -0,0 +1,1129 @@
+                                                <label class="form-check-label" for="workretired"><?php echo xlt('Retired');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="workdisabled" name='workdisabled' <?php echo (($obj["workdisabled"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="workdisabled"><?php echo xlt('Disabled');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="workstudent" name='workstudent'  <?php echo (($obj["workstudent"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="workstudent"><?php echo xlt('Student');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="worknotemployed" name='worknotemployed' <?php echo (($obj["worknotemployed"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="worknotemployed"><?php echo xlt('Not Employed Outside the Home');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="worknotans" name='worknotans' <?php echo (($obj["worknotans"] ?? '') == "on") ? "checked" : ""; ?>/>
+                                                <label class="form-check-label" for="worknotans"><?php echo xlt('Choose not to answer');?></label>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                        </div>
+                    </fieldset>
+                    <fieldset>
+                        <legend><?php echo xlt('How many hours do you work in a week?')?></legend>
+                        <div class="container">
+                            <div class="row">
+                                <div class="col-12">
+                                    <div class="form-group">
+                                        <div class="form-number">
+                                            <input type="number" id="workhours" name='workhours' min="0" max="200" value="<?php echo ($obj["workhours"] ?? ''); ?>"/>
```

Comment:
text() for html escaping (this prevents cross scripting (xss) attacks):
```
<input type="number" id="workhours" name='workhours' min="0" max="200" value="<?php echo text($obj["workhours"] ?? ''); ?>"/>
```

Reply:
goodness, losing my escaping touch :) for this and the other ones i mentioned below, will use attr()